src/m365/cli/commands/cli-issue.ts

```diff
@@ -0,0 +1,90 @@
+import { Logger } from '../../../cli';
+import Command, {
+  CommandOption
+} from '../../../Command';
+import GlobalOptions from '../../../GlobalOptions';
+import commands from '../commands';
+import * as open from 'open';
+import { ChildProcess } from 'node:child_process';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  type: string;
+}
+
+
+class CliIssueCommand extends Command {
```

Comment:
Because this command doesn't connect to M365, we should use the `AnonymousCommand` base class not to require users to sign in to M365 before using this command.